blocks/multi-panel-carousel/multi-panel-carousel.js

```diff
@@ -436,8 +436,8 @@ function addCarouselHeader(carouselContainer, title, dropdowns) {
       dropdownsDiv.appendChild(dropDownEle);
     });
     rowDiv.appendChild(dropdownsDiv);
-    document.addEventListener('click', (event) => {
-      closeAllDropDowns(event.target);
+    dropdownsDiv.addEventListener('click', (event) => {
```

Comment:
changed eventListener to `dropdownsDiv` element , same in the `i-lens` because of the same eventListener, they were interacting and were reacting with each other's events .

Reply:
Dropdown Experience is broken 
<img width="1394" alt="image" src="https://github.com/aemsites/icicidirect/assets/19485818/f65ed29b-2073-43de-a631-be5ea62265f0">

No two Dropdowns can be opened at same time. I will have a look

Reply:
changes reverted , we need here document level search to close all the drop down.